packages/content-conformance/src/runner.ts

```diff
@@ -50,14 +55,33 @@ export class ContentConformanceRunner {
     })
   }
 
+  /**
+   * TODO: Determine best ways to surface warnThreshold to user & default warnThreshold
+   */
   async run() {
-    return this.engine?.execute()
+    if (this.status === RunnerStatus.RUNNING) return null
+
+    // @ts-expect-error -- need to sort out VFile types here as well
+    const _statistics = statistics(this.engine?.files)
```

Comment:
This needs to be called after `this.engine?.execute()`, otherwise we will not have populated any messages.